<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1" colspan="2">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="4" column="1">
           <widget class="QListWidget" name="classList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QPushButton" name="btnQuery">
            <property name="text">
             <string>Query</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Classes</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QGroupBox" name="modeGroupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="radioButtonOpt">
               <property name="text">
                <string>Optimized</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonMobile">
               <property name="text">
                <string>Mobile</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonServer">
               <property name="text">
                <string>Server</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Process location</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QSpinBox" name="spinBoxK">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>13</number>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Top K</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>69</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>102</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>293</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>205</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Devices</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QStackedWidget" name="stackedDev1">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="dev1Off">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>No Device Connected</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="dev1On">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QFrame" name="frame">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="lineWidth">
                  <number>0</number>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <widget class="QLabel" name="dev1Label">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QTableView" name="dev1List">
                    <property name="font">
                     <font>
                      <pointsize>8</pointsize>
                     </font>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::NoSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                    <property name="gridStyle">
                     <enum>Qt::SolidLine</enum>
                    </property>
                    <attribute name="verticalHeaderVisible">
                     <bool>false</bool>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QStackedWidget" name="stackedDev2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="dev2Off">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>No Device Connected</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="dev2On">
              <layout class="QGridLayout" name="gridLayout_15">
               <item row="0" column="0">
                <widget class="QFrame" name="frame_5">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="lineWidth">
                  <number>0</number>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_16">
                  <item row="0" column="0">
                   <widget class="QLabel" name="dev2Label">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QTableView" name="dev2List">
                    <property name="font">
                     <font>
                      <pointsize>8</pointsize>
                     </font>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::NoSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                    <property name="gridStyle">
                     <enum>Qt::SolidLine</enum>
                    </property>
                    <attribute name="horizontalHeaderCascadingSectionResizes">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="verticalHeaderDefaultSectionSize">
                     <number>30</number>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QStackedWidget" name="stackedDev4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="dev4Off">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>No Device Connected</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="dev4On">
              <layout class="QGridLayout" name="gridLayout_17">
               <item row="0" column="0">
                <widget class="QFrame" name="frame_6">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="lineWidth">
                  <number>0</number>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_18">
                  <item row="0" column="0">
                   <widget class="QLabel" name="dev4Label">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QTableView" name="dev4List">
                    <property name="font">
                     <font>
                      <pointsize>8</pointsize>
                     </font>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::NoSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                    <property name="gridStyle">
                     <enum>Qt::SolidLine</enum>
                    </property>
                    <attribute name="verticalHeaderVisible">
                     <bool>false</bool>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QStackedWidget" name="stackedDev3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="dev3Off">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>No Device Connected</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="dev3On">
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="QFrame" name="frame_2">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="lineWidth">
                  <number>0</number>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_10">
                  <item row="0" column="0">
                   <widget class="QLabel" name="dev3Label">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QTableView" name="dev3List">
                    <property name="font">
                     <font>
                      <pointsize>8</pointsize>
                     </font>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::NoSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                    <property name="gridStyle">
                     <enum>Qt::SolidLine</enum>
                    </property>
                    <attribute name="verticalHeaderVisible">
                     <bool>false</bool>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
          <zorder>stackedDev1</zorder>
          <zorder>stackedDev3</zorder>
          <zorder>stackedDev2</zorder>
          <zorder>stackedDev4</zorder>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Results</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QLabel" name="matchLabel">
                <property name="text">
                 <string>Match</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTableView" name="resultsListMatch">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="gridStyle">
                 <enum>Qt::SolidLine</enum>
                </property>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="QLabel" name="noMatchLabel">
                <property name="text">
                 <string>No match</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTableView" name="resultsListNoMatch">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="gridStyle">
                 <enum>Qt::SolidLine</enum>
                </property>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelServer">
             <property name="text">
              <string>Server:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelMobile">
             <property name="text">
              <string>Mobile:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFP">
             <property name="text">
              <string>False positives:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFN">
             <property name="text">
              <string>False negatives:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelElapsedTime">
             <property name="text">
              <string>Elapsed time:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btnNewQuery">
          <property name="text">
           <string>New Query</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
